src/components/settings/ProvisioningSettings.vue

```diff
@@ -317,6 +317,7 @@
 						<Button
 							class="config-button save-config"
 							type="secondary submit"
+							@click="submitForm()"
```

Comment:
The line above seems wrong as well. It should just be `secondary`.

Reply:
For my understanding: my method is called submitForm(). how would the button know about it? do I have to change to method name to submit() instead? I iz confushon

Reply:
There is a `<form>` around the button. And if the button has the submit type then it will submit the form when clicked. And Vue will catch the form submit and call your method.

Reply:
:scream:

Reply:
yes, adding :native-type="submit" is enough, i did it with other buttons, but missed this one :)

Reply:
> (untested)

`:native-type` is wrong because it references the `submit` method. What I wanted to achieve is to pass the string `'submit'` to the `native-type` prop. https://github.com/nextcloud/mail/pull/7592 does that.